bibliopixel/led.py

```diff
@@ -18,9 +18,9 @@ def __init__(self, driver):
         self._data = []
         self._driver = driver
 
-    def setData(self, data):
+    def setData(self, colors, pos):
         self._reading.wait()
-        self._data = data
+        self._data = (colors, pos)
```

Comment:
Yeah... kinda one of the reasons I did the copy in the beginning. But wait... I have locking on there. Nevermind.